drivers/pds/lib/utils.go

```diff
@@ -1479,20 +1550,39 @@ func CreateDataServiceWorkloads(dataServiceName string, deploymentID string, sca
 	case redis:
 		env := []string{"REDIS_HOST", "PDS_USER", "PDS_PASS"}
 		command := "redis-benchmark -a ${PDS_PASS} -h ${REDIS_HOST} -r 10000 -c 1000 -l -q --cluster --user ${PDS_USER}"
-		pod, err = CreateRedisWorkload(deploymentName, redisStressImage, dnsEndpoint, pdsPassword, namespace, env, command)
+		pod, err = CreateRedisWorkload(params.DeploymentName, redisStressImage, dnsEndpoint, pdsPassword, params.Namespace, env, command)
 		if err != nil {
 			log.Errorf("An Error Occured while creating redis workload %v", err)
 			return nil, nil, err
 		}
 
 	case cassandra:
-		cassCommand := deploymentName + " write no-warmup n=1000000 cl=ONE -mode user=pds password=" + pdsPassword + " native cql3 -col n=FIXED\\(5\\) size=FIXED\\(64\\)  -pop seq=1..1000000 -node " + dnsEndpoint + " -port native=9042 -rate auto -log file=/tmp/" + deploymentName + ".load.data -schema \"replication(factor=3)\" -errors ignore; cat /tmp/" + deploymentName + ".load.data"
-		dep, err = CreatecassandraWorkload(cassCommand, deploymentName, namespace)
+		cassCommand := params.DeploymentName + " write no-warmup n=1000000 cl=ONE -mode user=pds password=" + pdsPassword + " native cql3 -col n=FIXED\\(5\\) size=FIXED\\(64\\)  -pop seq=1..1000000 -node " + dnsEndpoint + " -port native=9042 -rate auto -log file=/tmp/" + params.DeploymentName + ".load.data -schema \"replication(factor=3)\" -errors ignore; cat /tmp/" + params.DeploymentName + ".load.data"
+		dep, err = CreateDeploymentWorkloads(cassCommand, params.DeploymentName, cassandraStresImage, params.Namespace)
 		if err != nil {
 			log.Errorf("An Error Occured while creating cassandra workload %v", err)
 			return nil, nil, err
 		}
+	case elasticSearch:
+		esCommand := "while true; do esrally race --track=geonames --target-hosts=" + dnsEndpoint + " --pipeline=benchmark-only --test-mode --kill-running-processes --client-options=\"timeout:" + params.TimeOut + ",use_ssl:" + params.UseSSL + ",verify_certs:" + params.VerifyCerts + ",basic_auth_user:" + params.User + ",basic_auth_password:" + pdsPassword + "\"; done"
```

Comment:
use fmt.Sprintf that will clear and readable

Reply:
Done